pkg/apis/serving/v1beta1/predictor_triton.go

```diff
@@ -58,21 +58,21 @@ func (t *TritonSpec) Default(config *InferenceServicesConfig) {
 // GetContainers transforms the resource into a container spec
 func (t *TritonSpec) GetContainer(metadata metav1.ObjectMeta, extensions *ComponentExtensionSpec, config *InferenceServicesConfig) *v1.Container {
 	arguments := []string{
+		"trtserver",
```

Comment:
Shouldn't this be cmd?

Reply:
Can't use cmd here as that will overwrite the triton's entrypoint which does bunch of stuff
https://github.com/triton-inference-server/server/blob/master/nvidia_entrypoint.sh